test/testdata/bootstrappolicy/bootstrap_cluster_roles.yaml

```diff
@@ -64,6 +64,7 @@ items:
     - deploymentconfigrollbacks
     - deploymentconfigs
     - deploymentconfigs/log
+    - deploymentconfigs/rollback
```

Comment:
Be sure to verify that you're checking against NotAuthorized errors both when someone doesn't have the role (but rollback exists) and when this is an older server and doesn't have the endpoint, but you get a 401 because that's the default behavior for unsupported resources.